templates/batch-llm/README.ipynb

```diff
@@ -274,28 +228,89 @@
     "ds = ds.map_batches(\n",
     "    LLMPredictor,\n",
     "    # Set the concurrency to the number of LLM instances.\n",
-    "    concurrency=num_llm_instances,\n",
+    "    concurrency=4,\n",
     "    # Specify the number of GPUs required per LLM instance.\n",
-    "    num_gpus=num_gpus_per_instance,\n",
+    "    num_gpus=1,\n",
     "    # Specify the batch size for inference. Set the batch size to as large possible without running out of memory.\n",
-    "    # If you encounter CUDA out-of-memory errors, decreasing batch_size may help.\n",
+    "    # If you encounter out-of-memory errors, decreasing batch_size may help.\n",
     "    batch_size=10,\n",
+    "    # Pass keyword arguments for the LLMPredictor class.\n",
+    "    fn_constructor_kwargs={\"text_column\": \"item\"},\n",
     ")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Scaling with GPUs\n",
-    "\n",
-    "To use GPUs for inference in the Workspace, we specified `num_gpus` and `concurrency` in the `ds.map_batches()` call above to indicate the number of LLM instances and the number of GPUs per LLM instance, respectively. For example, with `concurrency=4` and `num_gpus=1`, we have 4 LLM instances, each using 1 GPU, so we need 4 GPUs total.\n",
-    "\n",
     "Finally, make sure to either enable *Auto-select worker nodes* or configure your workspace cluster to have the appropriate GPU worker nodes:\n",
     "\n",
     "![title](assets/ray-data-gpu.png)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Run the following cell to start dataset execution and view the results!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ds.take_all()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Scaling to a larger dataset\n",
```

Comment:
Similar comment below, if these are all under section 4 they need to be one level deeper as headings.